### PR TITLE
intacct-helper-doc

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html [hunk 1]
```diff
@@ -77,7 +77,16 @@ <h5 class="!tw-text-text-muted tw-text-14-px tw-pt-10-px !tw-font-400 !tw-leadin
                         <div class="inside-container">
                             <h4 class="tw-text-14-px !tw-font-500 tw-text-text-secondary tw-w-600-px">{{ error.error_title | snakeCaseToSpaceCase | titlecase }}</h4>
                             <div class="flex-wrapper">
-                                <h5 class="!tw-text-text-muted tw-text-12-px tw-pt-10-px !tw-font-400 !tw-leading-4 tw-w-600-px"><p>{{ error.error_detail }}</p>
+                                <h5 class="!tw-text-text-muted tw-text-12-px tw-pt-10-px !tw-font-400 !tw-leading-4 tw-w-600-px tw-flex tw-items-center tw-justify-start">
+                                    <p>{{ error.error_detail }}. &nbsp;
+                                        <a *ngIf="appName === AppName.INTACCT && brandingFeatureConfig.allowIntacctHelperDoc" class="tw-text-link-primary tw-w-fit tw-cursor-pointer tw-inline-flex tw-items-center"
```
Comment (ashwin1111) [inline src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html:82]: appName check is not required. add this to the condition - `brandingFeatureConfig.allowIntacctHelperDoc && error.article_link`

File: src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html [hunk 2]
```diff
@@ -77,7 +77,16 @@ <h5 class="!tw-text-text-muted tw-text-14-px tw-pt-10-px !tw-font-400 !tw-leadin
...
+                                            (click)="windowService.openInNewTab(error.article_link)">
+                                            Read More
```
Comment (ashwin1111) [inline src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html:84]: sentence case

File: src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html [hunk 3]
```diff
@@ -77,7 +77,16 @@ <h5 class="!tw-text-text-muted tw-text-14-px tw-pt-10-px !tw-font-400 !tw-leadin
...
+                                            <app-svg-icon *ngIf="brandingFeatureConfig.isIconsInsideButtonAllowed" [svgSource]="'open-in-new-tab'" [width]="'16px'" [height]="'16px'"
+                                                class="tw-text-link-primary tw-pl-2-px tw-w-fit">
+                                            </app-svg-icon>
+                                        </a>
+                                    </p>
```
Comment (ashwin1111) [inline src/app/shared/components/dashboard/dashboard-error-section/dashboard-error-section.component.html:89]: dialog change is missing

**Review by ashwin1111 — COMMENTED** [review]
pls also add screenshots

**Comment by DhaaraniCIT** [comment]
Screenshot 2024-05-20 at 10 03 48 AM

**Pull request merged**